recipes-core/images/nilrt-safemode-initramfs.bb

```diff
@@ -23,8 +24,6 @@ IMAGE_INSTALL_NODEPS += "\
 	${NI_PROPRIETARY_SAFEMODE_PACKAGES} \
 "
 
-BAD_RECOMMENDATIONS_append_pn-${PN} += "shared-mime-info"
-
 # Do not allow python to be installed into safemode ramdisk due to size
 PACKAGE_EXCLUDE += "python-core python3-core"
 
```

Comment:
There's an IMAGE_FSTYPES assignment down below that you may need to remove.

Reply:
No; the `IMAGE_FSTYPE = "cpio.xz"` assignment is correct for the safemode initramfs. It gets installed to the `nilrt-safemode-rootfs` as `runmode.xz`.

Reply:
Then, there is no need for the IMAGE_FSTYPES assignment from the `nilrt-image-initramfs.inc` includes file, and even less need for the includes file itself.